build.sbt

```diff
@@ -92,6 +92,59 @@ lazy val shocon = crossProject.in(file(".")).
 lazy val shoconJVM = shocon.jvm
 lazy val shoconJS = shocon.js
 
+lazy val plugin = project
+  .settings(
+    commonSettings ++
+    sonatypeSettings ++
+    publishingSettings: _*)
+  .settings(
+    name := "sbt-shocon",
+    description := "sbt plugin for shocon",
+    sbtPlugin := true,
+    scalaVersion := "2.10.6",
+    crossScalaVersions := Seq("2.10.6"),
+    addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion),
+    scalacOptions ++= Seq(
+      "-feature",
+      "-unchecked",
+      "-language:implicitConversions"),
+    // configuration for testing with sbt-scripted
+    ScriptedPlugin.scriptedSettings,
+    scriptedLaunchOpts ++= Seq("-Xmx1024M", "-Dplugin.version=" + version.value),
+    scriptedBufferLog := false,
+    publishLocal := publishLocal.dependsOn(publishLocal in shoconJS).dependsOn(publishLocal in shoconJVM).value
```

Comment:
could this be:
```
publishLocal.dependsOn(publishLocal in shocon).value
```
?
in case it is not maybe `dependsOn` takes multiple arguments